public/app/plugins/datasource/loki/language_provider.ts

```diff
@@ -67,13 +71,13 @@ export default class LokiLanguageProvider extends LanguageProvider {
     return syntax;
   }
 
-  request = (url: string) => {
-    return this.datasource.metadataRequest(url);
+  request = (url: string, params?: any) => {
```

Comment:
The request function of abstract class `LanguageProvider` in types/Explore.ts should be updated with `params` argument and probably string as `url` type.